regression/ansi-c/gcc_float_types1/main.c

```diff
@@ -0,0 +1,19 @@
+// for gcc, __float80 and __float128 are typedefs
+// for clang, __float128 is a keyword, and __float80 doesn't exist.
+
+#ifdef __clang__
+int __float80;
+__float128 f128;
+#else
+__float80 f80;
+__float128 f128;
+#endif
+
+int main()
+{
+  #ifndef __clang__
+  // on gcc, they can be re-declared, as they are identifiers, not keywords
```

Comment:
Perhaps run the test against Clang too, but checking for the expected syntax error in the .desc file?

Reply:
We don't have means to enable clang as preprocessor.

Reply:
goto-cc --native-compiler=clang should do so, I believe?

Reply:
Yes; are we willing to add clang as a dependency for CI for this?

Reply:
I'd say it's more an "in principle we could" case. We'll probably do if/when we run into a regression on this :-)